### ISSUE TITLE
[Bug]: Oil isn't spawning in on survival world | different world of same seed is spawning in oil just fine

### ISSUE THREAD

**Issue by onecrazybacon** [TeamGalacticraft/Galacticraft-Legacy#149]
### Forge Version

14.23.5.2854

### Galacticraft Version

4.0.5

### Reproduction steps

1. created survival world
2. noticed no oil was spawning
3. create identical world of same seed and noticed oil spawning in that world where the first world didn't

**Comment by ROMVoid95** [comment]
Going to need more information than this. Like a mod list (if anything else installed), logs, debug logs, anything

**Comment by onecrazybacon** [comment]
Alright I solved the issue and I know what caused it. I had some other mods installed including buildcraft which I accidentally left in my mods folder. I took it out after starting the world, I think what caused the issue is that buildcraft also has spawnable oil, when I removed buildcraft it also must've removed galacticraft's oil. I fixed the issue by removing galacticraft from my mods folder, booting up the game but not opening my world, closing the game, adding the mod back in, re-opening the game and my world and now oil is spawning on new chunks. Thanks for getting back to me regardless.
